spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -268,6 +272,7 @@
 "IBM Public License": IPL-1.0
 "ISC License (ISCL)": ISC
 "ISC/BSD License": ISC OR BSD-2-Clause
+"Indiana University Extreme! Lab Software License, version 1.1.1": LicenseRef-scancode-indiana-extreme
```

Comment:
You probably need to rebase due to @fviernau's fix of the typo in "LicenseRef-indiana-extreme-scancode" in the line below.